fontforge/splineoverlap.c

```diff
@@ -3613,9 +3605,7 @@ static SplineSet *SSRemoveTiny(SplineSet *base) {
 		refigure = true;
 	    }
 	    if ( sp->me.x==sp->prevcp.x && sp->me.y==sp->prevcp.y ) {
-		// We disable the control point if necessary.
-		sp->noprevcp = true;
-		if ((sp->prev) && (sp->prev->order2) && (sp->prev->from)) sp->prev->from->nonextcp = true;
+		if ((sp->prev) && (sp->prev->order2) && (sp->prev->from)) sp->prev->from->nextcp = sp->prev->from->me;
```

Comment:
If the outer guard equality holds and noprevcp is not already true it's because the value of the point or control point has changed, in which case `refigure` should be true.

Reply:
What about the similar `nonextcp` logic in the same function?

Reply:
Oh, wait. That was covered. I missed it somehow.